### PR TITLE
Added user_actual_id to rudder page event

### PR DESCRIPTION
**Pull request by marianunez**
#### Summary

Included user data under `user_actual_id`/`user_actual_role` to rudder page event. 

#### Ticket Link

https://mattermost.atlassian.net/browse/MM-29799

### CODE REVIEW

File: src/client/client4.ts [hunk 1]
```diff
@@ -3494,6 +3494,8 @@ export default class Client4 {
                 search: '',
                 title: '',
                 url: '',
```
Comment (marianunez) [inline src/client/client4.ts:3496]: @Adovenmuehle can you confirm these other props are ok for `page` events?

Comment (Adovenmuehle) [reply src/client/client4.ts:3496]: Yes this looks correct to me.

**Review by Adovenmuehle — APPROVED** [review]
Can you let me know when this gets to test cloud and I'll double check

**Review by devinbinnie — APPROVED** [review]

**Comment by marianunez** [comment]
> Can you let me know when this gets to test cloud and I'll double check

Sure, will ping you then!

**Pull request merged**